File: codes/WEIJIE/LC-232.java
class MyQueue {
    
    Stack <Integer> s1 = new Stack<>();
    Stack <Integer> s2 = new Stack<>();  
    
    /** Initialize your data structure here. */
    public MyQueue() {
        
    //Stack <Integer> s1 = new Stack<>();
    //Stack <Integer> stackTwo = new Stack<>();        
    }
    
    /** Push element x to the back of queue. */
    public void push(int x) {
        s1.push(x);
        
    }
    
    /** Removes the element from in front of queue and returns that element. */
    public int pop() {
        int size=0;
        while(!s1.empty()){
            //int y = s1.pop();
            //System.out.println(y);
            s2.push(s1.pop());
            size++;
        // System.out.println(size);
        }
        
        int temp = s2.pop();
        //System.out.println(temp);
        while(!s2.empty()){
            s1.push(s2.pop());
            }
        
         return temp;
    }
    
    /** Get the front element. */
    public int peek() {
        return s1.get(0);
    }
    
    /** Returns whether the queue is empty. */
    public boolean empty() {
        return s1.empty();
        
    }
}

File: Codes/Nancy/LC232.py
class MyQueue:

    def __init__(self):
        """
        Initialize your data structure here.
        """
        self.list = []

    def push(self, x: int) -> None:
        """
        Push element x to the back of queue.
        """
        self.list.append(x)

    def pop(self) -> int:
        """
        Removes the element from in front of queue and returns that element.
        """
        if len(self.list) > 0:
            return self.list.pop(0)
        else:
            return False
        

    def peek(self) -> int:
        """
        Get the front element.
        """
        if len(self.list) > 0:
            return self.list[0]
        else:
            return False        
        

    def empty(self) -> bool:
        """
        Returns whether the queue is empty.
        """
        return len(self.list) == 0
        


# Your MyQueue object will be instantiated and called as such:
# obj = MyQueue()
# obj.push(x)
# param_2 = obj.pop()
# param_3 = obj.peek()
# param_4 = obj.empty()